modules/olm-certificates.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificate-types-descriptions.adoc
+
+[id="olm-certificates_{context}"]
+= OLM certificates
+
+[discrete]
+== Management
+
+All certificates for OpenShift Lifecycle Manager (OLM) components
+(`olm-operator`, `catalog-operator`, `packageserver`, and
+`marketplace-operator`) are managed by the system.
+
+Operators installed via OLM can have certificates generated for them if they are
+providing API services. `packageserver` is one example.
+
+Certificates in the `openshift-operator-lifecycle-manager` namespace are managed
+by OLM with the exception of certificates used by Operators that require a
+validating or mutating webhook.
+
+Operators that install validating or mutating webhooks must currently manage
+those certificates themselves. They do not require the user to manage the
+certificates.
```

Comment:
Is this targeted at 4.4? This is changing with the introduction of [OLM support for admission webhooks in 4.5](https://github.com/operator-framework/operator-lifecycle-manager/pull/1436)